### PR TITLE
First draft of the nextep plugin using tvdb client library.

### PR DESCRIPTION
**Pull request by purdyk**
:boom:
http://i.imgur.com/F2IGWTO.gifv

### CODE REVIEW

File: quadlekBot.go [hunk 1]
```diff
@@ -64,10 +65,20 @@ func run(c *cli.Context) error {
...
 		return nil
 	}
 
+	if c.IsSet("tvdb-key") {
+		tvdbKey := c.String("tvdb-key")
+
+		err = bot.RegisterPlugin(nextep.Register(tvdbKey))
+		if err != nil {
+			fmt.Printf("error registering nextep plugin: %s", err.Error())
+			return nil
```
Comment (jirwin) [inline quadlekBot.go:78]: If you return here, it'll exit the `main()` function and exit the bot. Is that what you want?

Comment (purdyk) [reply quadlekBot.go:78]: Nope!

Comment (jirwin) [reply quadlekBot.go:78]: Ah never mind. I understand what is going on now.

Comment (purdyk) [reply quadlekBot.go:78]: Correction: Yep!

**Pull request merged**